lib/rucio/common/schema/belleii.py

```diff
@@ -56,7 +58,8 @@
 
 NAME = {"description": "Data Identifier name",
         "type": "string",
-        "pattern": r"^/[A-Za-z0-9\.\-\_\+\/]{1,%s}$" % NAME_LENGTH}
+        "maxLength": NAME_LENGTH,
+        "pattern": r"^/[A-Za-z0-9\.\-\_\+\/]+"}
```

Comment:
```suggestion
        "pattern": r"^/[A-Za-z0-9\.\-\_\+\/]+$"}
```

This fixes the test:

```
'invalid_name' does not match '^/[A-Za-z0-9\\.\\-\\_\\+\\/]+$'

Failed validating 'pattern' in schema['properties']['name']:
    {'description': 'Data Identifier name',
     'maxLength': 250,
     'pattern': '^/[A-Za-z0-9\\.\\-\\_\\+\\/]+$',
     'type': 'string'}

On instance['name']:
    'invalid_name'
'/belle/invalid@/did' does not match '^/[A-Za-z0-9\\.\\-\\_\\+\\/]+$'

Failed validating 'pattern' in schema['properties']['name']:
    {'description': 'Data Identifier name',
     'maxLength': 250,
     'pattern': '^/[A-Za-z0-9\\.\\-\\_\\+\\/]+$',
     'type': 'string'}

On instance['name']:
    '/belle/invalid@/did'
```

Reply:
Thanks!